libminifi/test/libtest/integration/IntegrationBase.h

```diff
@@ -31,26 +31,35 @@
 #include "utils/file/AssetManager.h"
 #include "utils/file/FileUtils.h"
 #include "core/BulletinStore.h"
+#include "unit/TestBase.h"
 
 namespace minifi = org::apache::nifi::minifi;
 namespace core = minifi::core;
 namespace utils = minifi::utils;
 
 namespace org::apache::nifi::minifi::test {
 
+struct FlowConfigPath {
+  std::unique_ptr<TempDirectory> temp_dir;
+  std::optional<std::filesystem::path> config_path;
+};
+
 class IntegrationBase {
  public:
-  explicit IntegrationBase(std::chrono::milliseconds waitTime = std::chrono::milliseconds(DEFAULT_WAITTIME_MSECS));
+  explicit IntegrationBase(const std::optional<std::filesystem::path>& test_file_location = {}, const std::optional<std::filesystem::path>& home_path = {},
```

Comment:
Does `test_file_location` (and `flow_config_path_.config_path`) still need to be an optional? We seem to set it everywhere. Could we do this?
```suggestion
  explicit IntegrationBase(const std::filesystem::path& test_file_location, const std::optional<std::filesystem::path>& home_path = {},
```

Reply:
OK, got it. I didn't see any in the diff, but I guess those tests did not require a change, so they're not in the diff.